content/8.x/admin/installation-guide/Upgrading.ru.adoc

```diff
@@ -0,0 +1,43 @@
+---
+Title: Обновление системы
+Weight: 30
+---
+
+:author: likhobory
+:email: likhobory@mail.ru
+
+
+:toc:
+:toc-title: Оглавление
+:toclevels: 1
+
+:experimental:
+
+:imagesdir: /images/ru/admin/Upgrading
+
+ifdef::env-github[:imagesdir: ./../../../../master/static/images/ru/admin/Upgrading]
+
+:btn: btn:
+
+ifdef::env-github[:btn:]  
+
+{{% notice note %}}
+Описание обновления для SuiteCRM 8 ещё в процессе создания, наберитесь терпения. 
+ {{% /notice %}}
+
+
+== Обновление системы
+
+В данном разделе описывается процесс обновления системы при помощи мастера обновления. 
+
+Прежде всего, необходимо скачать необходимый https://suitecrm.com/upgrade-suitecrm[пакет обновления^].
+ 
+{{% notice warning %}}
```

Comment:
Perhaps just a note for myself to update the english version to include this visual warning